tests/Console/SessionsCommandTest.php

```diff
@@ -18,9 +18,28 @@
 namespace Colopl\Spanner\Tests\Console;
 
 use Colopl\Spanner\Connection;
+use Colopl\Spanner\Session as SessionInfo;
+use Exception;
+use Google\Cloud\Spanner\Session\Session;
+use Google\Cloud\Spanner\Session\SessionPoolInterface;
 
 class SessionsCommandTest extends TestCase
 {
+    /**
+     * @param Connection $connection
+     * @param int $amount
+     * @return list<Session>
+     */
+    protected function createSessions(Connection $connection, int $amount): array
+    {
+        $pool = $connection->getSpannerDatabase()->sessionPool() ?? throw new Exception('unreachable');
```

Comment:
I think unchecked exception (or `RuntimeException` in this case) would be more appropriate for `unreachable' exceptions.

Reply:
I figured, this would suffice since it's a simple check to prevent phpstan from raising an erroring here.

In an optimal scenario, I would create a `UnreachableException` and use that, but that would be overkill for a simple check within a test. 

I have changed it to `RuntimeException ` per request.